ProcessLib/InitialCondition.h

```diff
@@ -57,7 +57,7 @@ class UniformInitialCondition : public InitialCondition
 
 /// Construct a UniformInitialCondition from configuration.
 std::unique_ptr<InitialCondition> createUniformInitialCondition(
-    BaseLib::ConfigTree const& config);
+    BaseLib::ConfigTree const& config, int const tuple_size);
```

Comment:
So the number of components of the IC has to match that of the process variable, I guess.
Makes sense.
Does it have to be documented?